src/features/transfer/TransferTokenCard.tsx

```diff
@@ -22,22 +19,7 @@ export function TransferTokenCard() {
             color={Color.primaryBlue}
           />
         </div>
-        {tokenRoutes && <TransferTokenForm tokenRoutes={tokenRoutes} />}
-        {isLoading && (
-          <div className="my-24 flex flex-col items-center">
-            <Spinner />
-            <h3 className="mt-5 text-sm text-gray-500">Finding token routes</h3>
-          </div>
-        )}
-        {routesError && (
-          <div className="my-32 flex flex-col items-center text-center">
-            <h3 className="text-red-500">Error searching for token routes.</h3>
-            <div className="mt-3 text-sm text-red-500">
-              Please ensure token configs are valid and RPCs are healthy.
-            </div>
-            <div className="mt-4 text-xs text-gray-500">{routesError.toString()}</div>
-          </div>
-        )}
+        <TransferTokenForm tokenRoutes={getWarpContext().routes} />
```

Comment:
Tbh we don't _really_ need to pass the routes around as props anymore but I'm gonna punt on refactoring the component props for now. If we ever do need async routes again, we'll be glad to have it structured this way